### PR TITLE
readme.md was added

### PR DESCRIPTION
**Pull request by Omrigotlieb**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Kickstart 2017 - github rotate repository
+
+[![N|Solid](http://downdetector.com/i/logo/Wix-Logo.png)](http://www.wixkickstart.com/)
```
Comment (danyshaanan) [inline readme.md]: Could you make this a bit smaller?

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+# Our github experiment!
+  - Rotate project
+  - Please contribute 
+
+> Our rotate repository
+> Please contribute
```
Comment (danyshaanan) [inline readme.md]: Are these two lines intentionally here? Seems similar to the two just above.

File: readme.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Kickstart 2017 - github rotate repository
 
-[![N|Solid](http://downdetector.com/i/logo/Wix-Logo.png)](http://www.wixkickstart.com/)
+<img src="http://downdetector.com/i/logo/Wix-Logo.png" alt="Drawing" style="width: 200px;"/>
```
Comment (danyshaanan) [inline readme.md]: Oh noes! Please don't use html 😢

**Comment by danyshaanan** [comment]
Oh noes! Please don't use html 😢

File: readme.md (hunk 2 above)
Comment (Omrigotlieb) [inline readme.md]: '>' character would act different and will add a line next to it. 
It can helps with the design of the .md file

Comment (danyshaanan) [inline readme.md]: I understand the markdown, but I do not understand why 'rotate project' and 'please contribute' appears twice.

**Comment by danyshaanan** [comment]
[Here](https://stackoverflow.com/questions/14675913/how-to-change-image-size-markdown) are instructions on how to resize a markdown image.

**Comment by Omrigotlieb** [comment]
The link you gave me for the image resize does not work for the external image.

**Comment by danyshaanan** [comment]
You can add the image in a `/docs/` directory. (I don't know where it's hosted, but I rather avoid hotlinking anyway).

**Pull request closed without merging**